### PR TITLE
minor: Moved the verifyAst UTs from AllBlockCommentsTest and AllSinglelineCommentsTest to AstTreeStringPrinterTest

### PR DESCRIPTION
**Pull request by subkrish**
This minor PR moves the `verifyAst` methods from `AllBlockCommentsTest` and `AllSinglelineCommentsTest` to `AstTreeStringPrinterTest`. This change would allow changes to be made to the above files in https://github.com/checkstyle/checkstyle/issues/4592 .

### CODE REVIEW

**Comment by rnveach** [comment]
@timurt Do you know what is wrong with wercker here?
> error: patch failed: checkstyle-tester/checks-nonjavadoc-error.xml:29
error: checkstyle-tester/checks-nonjavadoc-error.xml: patch does not apply

**Comment by rnveach** [comment]
@subkrish Try rebasing on latest master.

**Comment by subkrish** [comment]
@rnveach Done.

File: src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputFullOfBlockCommentsAst.txt [hunk 1]
```diff
@@ -5,206 +5,198 @@ BLOCK_COMMENT_BEGIN -> /* [1:5]
...
+|   |--BLOCK_COMMENT_BEGIN -> /* [1:101]
+|   |   |--COMMENT_CONTENT -> 10 [1:103]
+|   |   `--BLOCK_COMMENT_END -> */ [1:104]
 |   |--BLOCK_COMMENT_BEGIN -> /* [2:0]
-|   |   |--COMMENT_CONTENT -> 13 [2:2]
+|   |   |--COMMENT_CONTENT -> 11 [2:2]
 |   |   `--BLOCK_COMMENT_END -> */ [2:3]
 |   |--BLOCK_COMMENT_BEGIN -> /* [3:0]
-|   |   |--COMMENT_CONTENT -> 14 [3:2]
+|   |   |--COMMENT_CONTENT -> 12 [3:2]
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputFullOfBlockCommentsAst.txt]: Why did comment text change so much in this file? All you changed was package name on first line, right?

File: config/checkstyle_non_main_files_suppressions.xml [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
     <!-- Intentional no new line at the end. -->
      <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]grammars[\\/]comments[\\/]InputFullOfSinglelineComments\.java"/>
+     <suppress checks="NewlineAtEndOfFile" files="[\\/]test[\\/].*[\\/]astprinter[\\/]InputFullOfSinglelineComments\.java"/>
```
Comment (rnveach) [inline config/checkstyle_non_main_files_suppressions.xml]: shouldn't we change the first suppression instead of adding a 2nd?

Comment (subkrish) [inline config/checkstyle_non_main_files_suppressions.xml]: The reason is `AllSinglelineCommentsTest` still uses the input file in `testAllBlockComments()`.

File: src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputFullOfBlockCommentsAst.txt (hunk 1 above)
Comment (subkrish) [inline src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputFullOfBlockCommentsAst.txt]: That one change resulted in all of the input file to change. It was numbered and the numbering changed.

Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputFullOfBlockCommentsAst.txt]: > Old: /*0*//*1*/package/*2*/ com/*3*/./*4*/puppycrawl/*5*/./*6*/tools/*7*/./*8*/checkstyle.grammars/*9*/./*10*/comments/*11*/;/*12*/
> New: /*0*//*1*/package/*2*/ com/*3*/./*4*/puppycrawl/*5*/./*6*/tools/*7*/./*8*/checkstyle.astprinter/*9*/;/*10*/

Please change so numbering remains constant.
Example: /*0*//*1*/package/*2*/ com/*3*/./*4*/puppycrawl/*5*/./*6*/tools/*7*/./*8*/checkstyle/*9*/./*10*/astprinter/*11*/;/*12*/

File: config/checkstyle_non_main_files_suppressions.xml (hunk 2 above)
Comment (rnveach) [inline config/checkstyle_non_main_files_suppressions.xml]: I see now, the name of the files were starting to confuse me.
We can ignore this.

Comment (subkrish) [inline config/checkstyle_non_main_files_suppressions.xml]: Okay.

File: src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputFullOfBlockCommentsAst.txt (hunk 1 above)
Comment (subkrish) [inline src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputFullOfBlockCommentsAst.txt]: Done.

**Comment by subkrish** [comment]
@rnveach Requested changes have been made. Done. Please review.

**Comment by romani** [comment]
@subkrish , please rebase this change one more time, I just merged changes on javadoc tag names, your changes should be updated.

**Comment by subkrish** [comment]
@romani Rebased. Please review.

**Pull request merged**